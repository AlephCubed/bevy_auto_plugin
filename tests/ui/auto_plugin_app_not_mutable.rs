use bevy_auto_plugin_macros::*;

#[auto_plugin(app=_app)]
fn plugin(_app: &bevy::app::App) {}

// dummy main
fn main() {
    
}