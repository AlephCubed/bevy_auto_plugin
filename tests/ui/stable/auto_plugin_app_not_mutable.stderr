error: auto_plugin attribute must be used on a function with a `&mut bevy::app::App` parameter
 --> tests/ui/stable/auto_plugin_app_not_mutable.rs:4:11
  |
4 | fn plugin(_app: &bevy_app::App) {}
  |           ^^^^
